timesketch/lib/analyzers/domain.py

```diff
@@ -69,16 +70,28 @@ def run(self):
         satellite_emoji = emojis.get_emoji('SATELLITE')
         for domain, count in domain_counter.iteritems():
             emojis_to_add = [satellite_emoji]
+            tags_to_add = []
             text = '{0:s} seen {1:d} times'.format(domain, count)
 
+            is_known_cdn = utils.get_cdn_providers(domain)
```

Comment:
is_known_cdn sounds like a boolean to me... but it seems to be a list

Reply:
This is true. I have change this to return a string of the provider name instead. It should always be just one, but if for whatever reason there are multiple returned they will be joined to one string.